website/content/docs/platform/k8s/helm/terraform.mdx

```diff
@@ -0,0 +1,273 @@
+---
+layout: 'docs'
+page_title: 'Configure Vault Helm using Terraform'
+sidebar_current: 'docs-platform-k8s-terraform'
+description: |-
+  Describes how to configure the Vault Helm chart using Terraform
+---
+
+# Configuring Vault Helm with Terraform
+
+Terraform may also be used to configure and deploy the Vault Helm chart, by using the [Helm provider](https://registry.terraform.io/providers/hashicorp/helm/latest/docs).
+
+For example, to configure the chart to deploy [HA Vault with integrated storage (raft)](/docs/platform/k8s/helm/examples/ha-with-raft), the values overrides can be set on the command-line, in a values yaml file, or with a Terraform configuration:
+
+<CodeTabs>
+<CodeBlockConfig>
+
+```shell-session
+$ helm install vault hashicorp/vault \
+  --set='server.ha.enabled=true' \
+  --set='server.ha.raft.enabled=true'
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig>
+
+```yaml
+server:
+  ha:
+    enabled: true
+    raft:
+      enabled: true
+```
+
+</CodeBlockConfig>
+
+<CodeBlockConfig>
```

Comment:
```suggestion
<CodeBlockConfig heading="Terraform">
```

Would this work to replace HCL with Terraform? Feels marginally clearer to me, given that the Terraform config language is more specific than just HCL.

Reply:
Yeah I'd love to find a good way to replace HCL in the tab bar; setting `heading` ends up like this though:

<img width=340 src="https://user-images.githubusercontent.com/444265/181127161-0d7ccb56-dc9a-4908-9431-54926ef9888f.png"/>

Setting the code block to be terraform works (```terraform), but then we lose the syntax highlighting.

Reply:
Fair, I think as you have it now is the best then 👍